stackslib/src/chainstate/stacks/boot/signers.clar

```diff
@@ -0,0 +1,22 @@
+(define-data-var last-set-cycle uint u0)
+(define-data-var stackerdb-signer-slots (list 4000 { signer: principal, num-slots: uint }) (list))
+
+(define-private (stackerdb-set-signer-slots 
+                   (signer-slots (list 4000 { signer: principal, num-slots: uint }))
+                   (reward-cycle uint))
+	(begin
+		(print signer-slots)
```

Comment:
Just FYI that as far as I know, no (print) events were established. I added this print myself so it might not be necessary.

Reply:
Removed for now